packages/react-component-library/src/components/ThemedExample/ThemedExample.stories.tsx

```diff
@@ -12,7 +12,6 @@ export default {
   title: 'Custom Theming',
   parameters: {
     a11y: {
-      disabled: true,
```

Comment:
`disabled` is deprecated (`disable` is the replacement), but on closer inspection this was disabling the a11y addon and disabling one of its rules. Both aren't necessary, so I've left it at disabling the failing rule only.